dashboard/app/models/pd/workshop.rb

```diff
@@ -396,10 +396,10 @@ def end!
   # for details.
   def self.process_ends
     end_on_or_after(Time.now - 2.days).each do |workshop|
-      if !workshop.processed_at || workshop.processed_at < workshop.ended_at
-        workshop.send_exit_surveys
-        workshop.update!(processed_at: Time.zone.now)
-      end
+      next unless !workshop.processed_at || workshop.processed_at < workshop.ended_at
```

Comment:
I always struggle with `unless` in these more complicated conditionals.  Would a comment help explain when we skip here?

Reply:
will add a comment (the reason I changed it is the linter complained)